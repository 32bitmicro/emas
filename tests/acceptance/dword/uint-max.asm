.dword 4294967295
