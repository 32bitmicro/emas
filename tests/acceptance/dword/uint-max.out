@ 0x0000 : 0xffff  /  111 111 1 111 111 111  /  65535
@ 0x0001 : 0xffff  /  111 111 1 111 111 111  /  65535
